### ISSUE TITLE
Colchicine worsens symptoms

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11082]
**Preferred term label:**
Colchicine worsens symptoms

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
colchicine myopathy

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001324
- label: Exacerbated by colchicine
- parents: [Aggravated by exposure to medication (HP:4000176)](https://hpo.jax.org/app/browse/term/HP:4000176)
- pmid: 
- def: Applied to a sign or symptom that is worsened by treatment with colchizine.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11082
- synonyms: Colchicine worsens symptoms(NONE)

**Comment by MickeySegal** [comment]
Typo *colchizine*
